Src/Sarif.PatternMatcher.Function/HttpAnalyzeFn.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Function
+{
+    public static class HttpAnalyzeFn
+    {
+        [FunctionName("analyze-file")]
+        public static async Task<IActionResult> AnalyzeFile(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = null)] HttpRequest req,
+            ILogger log,
+            ExecutionContext context)
+        {
+            log.LogInformation("C# HTTP trigger function processed a request.");
```

Comment:
>LogInformation [](start = 16, length = 14)

Generally, putting entirely static strings in a log is less helpful than emitting strings that provide per-request details. Will the logger itself provide information on, for example, a request id (I assume  it will add a time-stamp). Can you provide an example of log output for us to validate?

Reply:
if we run it locally, it will show in the console (if im not mistaken). if we add AppInsights, it will log there. if we dont plug anything, it would only be available if we look in azure function portal, monitoring

---
In reply to: [553654570](https://github.com/microsoft/sarif-pattern-matcher/pull/85#discussion_r553654570) [](ancestors = 553654570)